src/main/java/com/textmessenger/model/Message.java

```diff
@@ -0,0 +1,26 @@
+package com.textmessenger.model;
+
+import lombok.Data;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "messages")
+@Data
+public class Message {
+
+  @Id
+  private long id;
+
+  @Column(name = "fromUser")
```

Comment:
I advise you to use lowercase only names for DB fields separated by underscores - this way it will work the same with every DB implementation